src/modules/components/proposal/proposalActions/actions/generators/proposalActionWithdrawToken.ts

```diff
@@ -3,11 +3,11 @@ import { ProposalActionType, type IProposalActionWithdrawToken } from '../../pro
 export const generateToken = (
     token?: Partial<IProposalActionWithdrawToken['token']>,
 ): IProposalActionWithdrawToken['token'] => ({
-    name: 'Dai Stablecoin',
-    symbol: 'DAI',
-    logo: 'https://cryptologos.cc/logos/multi-collateral-dai-dai-logo.png',
+    name: 'Ether',
```

Comment:
I'd avoid updating the `generateToken` generator because we need some specific data on the stories, I'd just update the data on the story instead